### PR TITLE
yugioh file, icon, route

### PR DESCRIPTION
**Pull request by krymarie**

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -2,12 +2,14 @@ import Home from './components/Home';
 import SignUp from './views/SignUp';
 import SignIn from './views/SignIn';
 import Yahtzee from './components/Yahtzee';
-import Nertz from './views/games/Nertz';
+import Nertz from './views/Nertz';
```
Comment (skaduusch) [inline src/routes.js]:
```suggestion
import Nertz from './views/games/Nertz';
```

Comment (krymarie) [reply src/routes.js]: thanks

File: src/routes.js [hunk 2]
```diff
@@ -3,11 +3,13 @@ import SignUp from './views/SignUp';
 import SignIn from './views/SignIn';
 import Yahtzee from './components/Yahtzee';
 import Nertz from './views/games/Nertz';
```
Comment (krymarie) [inline src/routes.js]: It's still there?

Comment (skaduusch) [reply src/routes.js]: New error is the empty <template></template> tag in your vue file

Comment (krymarie) [reply src/routes.js]: saw that, added a div in there should fix it

Comment (krymarie) [reply src/routes.js]: I had some files not tracking.

**Pull request merged**